schema/deploy/tables/funding_stream.sql

```diff
@@ -0,0 +1,41 @@
+-- Deploy cif:tables/funding_stream to pg
+
+BEGIN;
+
+create table cif.funding_stream(
+  id integer primary key generated always as identity,
+  name varchar(1000) not null,
+  description varchar(10000) not null
+);
+
+select cif_private.upsert_timestamp_columns('cif', 'funding_stream');
+create unique index funding_stream_name on cif.funding_stream(name);
+
+do
+$grant$
+begin
+
+-- Grant cif_internal permissions
+perform cif_private.grant_permissions('select', 'funding_stream', 'cif_internal');
+perform cif_private.grant_permissions('insert', 'funding_stream', 'cif_internal');
+perform cif_private.grant_permissions('update', 'funding_stream', 'cif_internal');
+
+-- Grant cif_admin permissions
+perform cif_private.grant_permissions('select', 'funding_stream', 'cif_admin');
+perform cif_private.grant_permissions('insert', 'funding_stream', 'cif_admin');
+perform cif_private.grant_permissions('update', 'funding_stream', 'cif_admin');
+
+-- Grant cif_external no permissions
+-- Grant cif_guest no permissions
+
+end
+$grant$;
+
+comment on table cif.funding_stream is 'Table containing information about a CIF funding_stream';
+comment on column cif.funding_stream.id is 'Unique ID for the funding_stream';
+comment on column cif.funding_stream.name is 'Shortname of the funding stream program';
+comment on column cif.funding_stream.description is 'Description of the funding_stream';
+
+insert into cif.funding_stream (name, description) values ('IA', 'Innovation Accelerator'), ('EP', 'Emissions Performance');
```

Comment:
we need EP to be id = 1 and IA to be id = 2, that's how the business refers to them right now - and how it's being used to generate the unique project IDs

Reply:
I swapped the order so EP will have id = 1 and IA id = 2